app/src/controllers/log/reducer.js

```diff
@@ -0,0 +1,27 @@
+import { combineReducers } from 'redux';
+import { fetchReducer } from 'controllers/fetch';
+import { CHANGE_ACTIVE_LOG_ITEM, SET_LOG_ITEM_TO_GET_HISTORY, NAMESPACE } from './constants';
+
+export const activeItemReducer = (state = '', { type, payload }) => {
+  switch (type) {
+    case CHANGE_ACTIVE_LOG_ITEM:
+      return payload;
+    default:
+      return state;
+  }
+};
+
+export const itemToGetHistoryReducer = (state = '', { type, payload }) => {
+  switch (type) {
+    case SET_LOG_ITEM_TO_GET_HISTORY:
+      return payload;
+    default:
+      return state;
+  }
+};
+
+export const logReducer = combineReducers({
+  logEntries: fetchReducer(NAMESPACE),
```

Comment:
I think it's better to name it `historyEntries` since it's actually history. Sorry for misleading in my previous review.

Reply:
Fixed